Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.py

```diff
@@ -0,0 +1,1479 @@
+        'ok' if the connection was successful, else throws exception.
+    """
+    body: Dict = {}
+    if title:
+        body["title"] = title
+    if source_branch:
+        body["source"] = {
+            "branch": {
+                "name": source_branch
+            }
+        }
+    if destination_branch:
+        body["destination"] = {
+            "branch": {
+                "name": destination_branch
+            }
+        }
+    if reviewer_id:
+        reviewers_arr = reviewer_id.split(',')
+        account_id_list = []
+        for id in reviewers_arr:
+            dict = {
+                "account_id": id
+            }
+            account_id_list.append(dict)
+        body["reviewers"] = account_id_list
+    if description:
+        body["description"] = description
+    if close_source_branch:
+        body["close_source_branch"] = argToBoolean(close_source_branch)
+    return body
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """ Test the connection to bitbucket.
+    Args:
+        client: A Bitbucket client.
+    Returns:
+        'ok' if the connection was successful, else throws exception.
+    """
+    try:
+        client.get_project_list_request(page=1, page_size=1)
+        return "ok"
+    except DemistoException as e:
+        raise Exception(e.message)
+
+
+def project_list_command(client: Client, args: Dict) -> CommandResults:
+    """ Returns a list of all the projects in Bitbucket.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A CommandResult object with a success message, in case of a successful action.
+    """
+    arg_limit = arg_to_number(args.get('limit', 50))
+    limit = arg_limit if arg_limit else 50
+    project_key = args.get('project_key')
+    page = arg_to_number(args.get('page', 1))
+    page_size = min(limit, 100)
+    response = client.get_project_list_request(page, page_size, project_key)
+
+    if project_key:
+        results = [response]
+        readable_name = f'The information about project {project_key.upper()}'
+    else:
+        results = check_pagination(client, response, limit)
+        readable_name = f'List of projects in {client.workspace}'
+
+    human_readable = []
+    key_list = []
+
+    for value in results:
+        d = {'Key': value.get('key'),
+             'Name': value.get('name'),
+             'Description': value.get('description'),
+             'IsPrivate': value.get('is_private')}
+        human_readable.append(d)
+        key_list.append(value.get('key'))
+
+    headers = ['Key', 'Name', 'Description', 'IsPrivate']
+
+    readable_output = tableToMarkdown(
+        name=readable_name,
+        t=human_readable,
+        removeNull=True,
+        headers=headers
+    )
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Bitbucket.Project',
+        outputs=results,
+        raw_response=results,
+        outputs_key_field=key_list
```

Comment:
This should be a unique field not the entire list - in this case it will be Key
```suggestion
        outputs_key_field=Key
```